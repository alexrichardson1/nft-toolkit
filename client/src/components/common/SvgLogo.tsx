interface PropsT {
  icon: string;
  width: string | number;
  height: string | number;
}

const SvgLogo = (props: PropsT): JSX.Element => (
  <img width={props.width} height={props.height} src={props.icon} />
);

export default SvgLogo;
