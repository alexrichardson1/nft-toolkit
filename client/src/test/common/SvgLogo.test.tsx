import SvgLogo from "components/common/SvgLogo";
import { mount } from "enzyme";
import EnzymeToJson from "enzyme-to-json";
import AvalancheLogo from "images/avalanche-logo.svg";

test("SvgLogo snapshot", () => {
  const toJson = EnzymeToJson;
  const tree = mount(
    <SvgLogo icon={AvalancheLogo} width={"50"} height={"50"} />
  );

  expect(toJson(tree)).toMatchSnapshot();
});
