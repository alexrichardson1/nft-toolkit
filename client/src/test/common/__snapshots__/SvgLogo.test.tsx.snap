// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgLogo snapshot 1`] = `
<SvgLogo
  height="50"
  icon="avalanche-logo.svg"
  width="50"
>
  <img
    height="50"
    src="avalanche-logo.svg"
    width="50"
  />
</SvgLogo>
`;
